src/main/java/com/thebois/models/world/World.java

```diff
@@ -175,7 +176,17 @@ private boolean isPositionPlaceable(final Position position) {
      *
      * @return the colony.
      */
-    public IBeingGroup getColony() {
+    public AbstractBeingGroup getColony() {
+        return colony;
+    }
```

Comment:
question

Is this getter necessary anymore, since you added the `getColonyInventory` method?

Reply:
Yes, `ColonyController` is using it

Reply:
Created a task of what we talked about yesterday, about moving the creation of colony to `ColonyManagment`, but should I just do it in this userstory?

Reply:
If it's not too much work then that'd be great, one less task to do later on. But if it's a bit of a hassle then it's probably better to leave it for later. Up to you.